lib/projectLink.js

```diff
@@ -0,0 +1,13 @@
+class ProjectLink {
+  constructor(project, label, url) {
+    this.project = project;
+    this.label = label;
+    this.url = url;
+  }
+
+  async save(db) {
+    await db.addLink(this.project.channel_id, this.label, this.url);
```

Comment:
I think this is showing a refactor that should take place. We are leaking the `channel` information out of the `Project` class. I would recommend we have `Project.id` as our name for the id. That we are using the channel id as the `Project.id` is internal information to the `Project` class.